site/docs/v1/tech/guides/breached-password-detection.adoc

```diff
@@ -0,0 +1,274 @@
+
+The end goal, of course, is to have a more secure auth system by avoiding unauthorized access due to any of your users' credentials being compromised.
+
+== How Do I Use Breached Password Detection 
+
+*This is a paid edition feature and is unavailable in the community edition. Please visit link:/pricing[our pricing page] to learn more about paid editions.*
+
+Here's a video showing setup and usage of the breached password detection feature.
+
+video::JUB-5wQ_fdI[youtube,width=560,height=315]
+
+Conceptually, FusionAuth uses a pull model rather than a push model. Passwords are checked only when the user provides, at registration, password change, or, optionally, during login. 
+
+== Setting Up
+
+The first step to enabling breached password detection is to have a valid license key. Please visit link:/pricing[our pricing page] to learn more about paid editions and to buy a license.
+
+The next step is to activate your license key. You'll need to ensure that your FusionAuth instance has outbound network access. Follow the steps outlined in the link:/docs/v1/tech/reactor[Reactor documentation].
+
+Navigate to the [breadcrumb]#Tenants# tab and edit the tenant for which you wish to enable breached password detection. Go to the [breadcrumb]#Password# tab and the [breadcrumb]#Breached password detection settings# section. Click the checkbox to enable breached password detection. This will open up a new set of configuration options, as seen below:
+
+image::guides/breached-password-detection/turn-on-detection.png[Turn on breached password detection,width=700,role=box-shadow margin-right]
+
+The default settings prevent use of leaked passwords in a number of contexts. Simply by enabling breached password detection, all users within this tenant will, from that moment forward, no longer be able to use any password that has been found to be breached by the FusionAuth team.
+
+Passwords will be checked any time they are changed, such as:
+
+* at user registration
+* when a password change initiated by an end user
+* a password changes made by an administrative user
+
+You can see below that an administrative user will receive an error message should they attempt to change an end user's password to a value known to be breached:
+
+image::guides/breached-password-detection/admin-user-attempting-to-use-breached-password.png[Error message when an administrative user tries to use a compromised password.,width=700,role=box-shadow margin-right]
+
+However, if an administrative user changes any other user attribute without modifying the password, the current password won't be checked.
+
+Password changes, whether triggered by a user interacting with the OAuth registration pages or an application using the API, will cause the password to be checked against the latest corpus of compromised passwords. If there is a match, an error will be returned.
```

Comment:
Registration pages? Seems like this would be "themed pages", or "login pages". 

Themed pages where user can enter a password would be registration (create user + registration), login, and change password.